src/css/ds-blocks.css

```diff
@@ -3,77 +3,135 @@
 /*
  *  Usage: [source,python]
  *
- *  Default style: [source.default,python]
+ *  Text wrap style: [source.wrap,python]
+ *  Hide language title style: [source.hidelang,python]
  */
-
-.listingblock:not(.default) .content {
-  display: grid;
+.doc code.hljs {
+  color: var(--ds-text-secondary);
 }
 
-.listingblock:not(.default) .content .source-toolbox {
+.doc .highlightjs {
   position: relative;
-  padding: var(--ds-space-1) var(--ds-space-2);
-  visibility: visible;
-  justify-content: flex-end;
-  background: var(--ds-neutral-100);
-  grid-row-start: 1;
+}
+
+.doc .listingblock:not(.wrap) .highlightjs {
+  text-wrap: nowrap;
+}
+
+/* .doc .source-titlebar {
+  padding: var(--ds-space-h) var(--ds-space-1);
+  background: var(--ds-background-level2);
   border-top-left-radius: calc(6 / var(--rem-base) * 1rem);
   border-top-right-radius: calc(6 / var(--rem-base) * 1rem);
-  top: 0;
-  left: 0;
+  border: 1px solid transparent;
+} */
```

Comment:
can we remove the commented out blocked?

Reply:
Yep!